source/connection-strings/connection-strings.txt

```diff
@@ -2,6 +2,16 @@
 Connection Strings
 ==================
 
```

Comment:
(wording)
It looks like this page didn't really get a refactor. I'll make a few rewrite suggestions here. 
Suggest rewriting first para. Something like:
"Relational Migrator uses connection strings to connect to both the relational and MongoDB database. You can create connection strings:

- Using the UI when you create a project.
- Using the UI when you start a sync job. 
- Manually creating URIs. 

To learn how to manually create your connection string URIs, see:

-  ... "